doc/tags_old.txt

```diff
@@ -1147,7 +1147,7 @@ v1_6	The XML file now supports comments.
 	Added support for CCB.
         Monitoring refactored
           Most rrd operations now in separate module
-          Groupes multiple attributes in the RRD files (lower overhead)
+          Grouped multiple attributes in the RRD files (lower overhead)
```

Comment:
`Grouped` or `Groups` (not sure which is relevant in the context)?

Reply:
I think Grouped - action in Monitoring refactored